rsconnect/actions_content.py

```diff
@@ -172,13 +194,28 @@ def build_start(connect_server, parallelism, aborted=False, error=False, all=Fal
             exit(1)
     except KeyboardInterrupt:
         ContentBuildStore._BUILD_ABORTED = True
+        logger.info("Content build interrupted...")
+        logger.info(
+            "Content that was in the RUNNING state may still be building on the "
+            + "Connect server. Server builds will not be interrupted."
+        )
+        logger.info(
+            "To find content items that _may_ still be running on the server, "
+            + "use: rsconnect content build ls --status RUNNING"
+        )
+        logger.info(
+            "To retry the content build, including items that were interrupted "
+            + "or failed, use: rsconnect content build run --retry"
+        )
     finally:
+        # make sure that we always mark the build as complete but note
+        # there's no guarantee that the content_executor or build_monitor
+        # were allowed to shut down gracefully, they may have been interrupted.
+        _content_build_store.set_build_running(False)
```

Comment:
with this setup, its still possible to ^C into a weird state, right? Just significantly less likely if we set the state of the build first thing.

Reply:
Yep, that's right. I'm hoping this covers most cases. It could still happen on slow filesystems but this makes it less likely in most scenarios. If the problem persists after this change then we can look at removing the safety check.

The reason it exists in the first place is to stop users from modifying the state file while there's a build running. The 3 commands that modify the local start are `content build [run, add, rm]`.

Reply:
yeah, I completely agree with the approach here - just making sure I am following what's going on.